ansible/ci/test_reimage.yml

```diff
@@ -63,3 +69,39 @@
     - name: Check slurm up after reimaging login node
       import_tasks: "{{ lookup('env', 'APPLIANCES_ENVIRONMENT_ROOT') }}/hooks/check_slurm.yml"
       run_once: true
+
+- hosts: control:!builder
+  become: no
+  gather_facts: no
+  tags: reimage_control
+  tasks:
+    - name: Reimage control node via openstack
+      shell:
+        cmd: "openstack server rebuild {{ instance_id | default(inventory_hostname) }} --image {{ control_build.artifact_id }}"
+      delegate_to: localhost
+    
+    - name: Check control node rebuild completed
+      shell:
+        cmd: openstack server show {{ inventory_hostname }} --format value -c image
+      register: openstack_control
+      delegate_to: localhost
+      retries: 5
+      delay: 30
+      until: control_build.artifact_id in openstack_control.stdout
+      changed_when: false
+    
+    - name: Wait for control connection
+      wait_for_connection:
+        timeout: 800
+    
+- name: Run slurm playbook again to add partition info
```

Comment:
Will this bit be needed when state is offboarded? Wondering out of my own curiosity really!

Reply:
Currently, yes. The [list of things offboarded](https://github.com/stackhpc/ansible-slurm-appliance/pull/173#issue-1210971616) doesn't include /etc/slurm.conf. Maybe it should do though, bit of a blurry line between "state" and "config" ...